includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-courses.php

```diff
@@ -57,21 +57,51 @@ public function get_items( array $filters ): array {
 			'suppress_filters' => 0,
 		);
 
+		if ( ! empty( $filters['orderby'] ) && ! empty( $filters['order'] ) ) {
+			$course_args['orderby'] = $filters['orderby'];
+			$course_args['order']   = $filters['order'];
+		}
+
 		if ( isset( $filters['search'] ) ) {
 			$course_args['s'] = $filters['search'];
 		}
 
 		add_filter( 'posts_clauses', [ $this, 'filter_courses_by_last_activity' ] );
 		add_filter( 'posts_clauses', [ $this, 'add_days_to_completion_to_courses_queries' ] );
+
+		if ( isset( $course_args['orderby'] ) && ( 'count_of_completions' === $course_args['orderby'] ) ) {
```

Comment:
I think you can safely assume that the `$course_args['orderby']` is already set.

Reply:
[Removed it](https://github.com/Automattic/sensei/pull/4970/commits/a411bd74121681db267ee86ce3838b70265feec6)